src/Settings/Builders/HsCpp.hs

```diff
@@ -1,6 +1,8 @@
 module Settings.Builders.HsCpp (hsCppBuilderArgs) where
 
 import Settings.Builders.Common
+import Builder ()
```

Comment:
Why is this needed?

Reply:
Because GHC wanted it there I believe, I'll try to drop it again, and see if GHC complains again.